doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
-1. Provide ability for any number of Operator-specific fabrics to power CNTT cloud infrastructure
-1. Enable RC's ability to realize mandated OVP qualification deliverables
-1. Unambiguously document the responsibilities of each CNTT constituency
-1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
-1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
-1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
-
-> _**Editor's Note:** Consider moving objectives to a table_
+### 5.1.2 Networking Principles & Strategy
+
+This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
+
+•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+> **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+-	Control, User and Management traffic types
+-	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+
+•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+
+•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•	**Workload diversity**: Should be capable to handle any VNF , CNF and BareMetal workloads.
+
+•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
+
+•	**Future proof**: Should support known and emerging acceleration technologies e.g. SmartNICs, FPGAs and Programmable Switches. 
+
```

Comment:
Should we include multi-cloud (public/private/hybrid),  multi-location (multi-cluster, DCI, etc), Edge?

Reply:
@sukhdevkapur good commentes, thanks.  I have added more examples along the lines of your advice.